chain/network/src/peer/peer_actor.rs

```diff
@@ -1029,7 +1030,8 @@ impl PeerActor {
                     Self::receive_routed_message(
                         &clock,
                         &network_state,
-                        peer_id,
+                        msg.msg.author,
+                        peer_id.clone(),
```

Comment:
This part is a bit confusing, why do we need a `.clone()` now if we did not before?